relayer/crates/starknet-chain-components/src/impls/types/client.rs

```diff
@@ -42,7 +65,12 @@ where
         false
     }
 
-    fn client_state_has_expired(_client_state: &Chain::ClientState, _elapsed: Duration) -> bool {
+    fn client_state_has_expired(_client_state: &Chain::ClientState, elapsed: Duration) -> bool {
+        // TODO: Properly define how to identify expired client. To simulate passing test uncomment
+        // this since client recovery test uses a trusting period of 40 seconds.
+        //if elapsed.as_secs() > 40 {
+        //    return true;
+        //}
```

Comment:
why this just returns false ? (assuming this is for tendermint light client), should we compare with the trusting period to calculate this ?

Reply:
Done in 88d70446de1184817d535f753ddcb99075a0d5c9